### ISSUE TITLE
Enhancing Keptn's functionality to change the password via the CLI

### ISSUE THREAD

**Issue by warber** [keptn/keptn#4435]
# User Story
Users of Keptn shall be able to change their Keptn Bridge password only when providing the recent password

## Details
This issue is security-related. It should not be possible to change the password without providing the recent password. the Keptn CLI shall provide a prompt like the possibility to change the password without showing the actual password typed in by the user: 

``` 
# change password with options
$ ~ keptn configure bridge --user=<user> --password=<pwd> --old-passwrod=<old-pwd>
Bridge credentials configured successfully

# change password with user prompt (user input shall not be shown. if new password confirmation fails then message is shown to the user and password is not changed)

$ ~ keptn configure bridge --user keptn
Please provide your current password: <userinput>
Please provide your new password: <userinput>
Please confirm your new password: <userinput>

Bridge credentials configured successfully
```

## Acceptance Criteria
The user is able to change the password like it is described above

## Definition Of Done
- [ ] Add possibility to change the old password
- [ ] Documentation updated and tell how to read the secret with `kubectl`
- [ ] CLI Installation request for the default password
- [ ] Integration Test available
- [ ] Tested CLI on Linux/Windows/MacOS

~:question: Open question: How to handle `keptn configure bridge --output` which returns the current password?
This behavior would bypass the above feature and make it useless. If we stop providing this command the user has the
problem of initially knowing the credentials to access the Keptn Bridge.~

---
## Resources
Hide user input in GO: https://stackoverflow.com/questions/2137357/getpasswd-functionality-in-go

**Comment by johannes-b** [comment]
To be clarified: The API endpoint and Keptn CLI are protected by an API token. Can the API token (which is secured and not accessible) be considered like providing the "old password"?

**Comment by thisthat** [comment]
The open question will be resolved by #4589

**Comment by agrimmer** [comment]
If a user configured Bridge to use OAuth instead of Basic Auth for the authentication, the endpoint for changing the password should be completely removed.
